tools/xcodeproj_shims/build-wrapper.sh

```diff
@@ -16,6 +16,7 @@ fi
 
 if [ $BAZEL_PROFILE_ENABLED -gt 0 ]; then
     BAZEL_BUILD_OPTIONS+=("--profile=$BAZEL_PROFILE_FILENAME")
+    BAZEL_BUILD_OPTIONS+=("--starlark_cpu_profile=$BAZEL_STARLARK_CPU_PROFILE_FILENAME")
```

Comment:
I think my concern here is that the `--profile` is always generated even if we don't specify it via `--profile` so we're not introducing overhead by adding it.

That's different with `--starlark_cpu_profile` though, IIRC it's not always created by Bazel. Meaning this will introduce additional overhead (even if minimal depending on the workspace). But still, I think this should be separate opt-in behaviour instead of being part of `BAZEL_PROFILE_ENABLED`. Thoughts?

Reply:
I agree. This should definitely be opt-in. I can create a `BAZEL_STARLARK_CPU_PROFILE_ENABLED` flag to control that.